protractor_test/index.bzl

```diff
@@ -16,8 +16,8 @@ def protractor_test(name, server, deps, extra_config = {}, data = [], enable_per
             ":%s_protractor_extra_config" % name,
             "@rules_browsers//browsers/chromium",
             "@rules_browsers//protractor_test:config",
-            Label("//:node_modules/protractor"),
-            Label("//:node_modules/tinyglobby"),
+            "//:node_modules/protractor",
+            "//:node_modules/tinyglobby",
```

Comment:
is this no longer needed now? If the macro is invoked from e.g. `angular` workspace, these labels will be incorrect right?